Applications/Accessories/MobilePASS/Online/script.js

```diff
@@ -0,0 +1,11 @@
+const OnlineInstallerScript = include("engines.wine.quick_script.online_installer_script");
+
+new OnlineInstallerScript()
+    .name("SafeNet MobilePASS")
+    .editor("SafeNet")
+    .applicationHomepage("https://safenetmp.com/")
+    .author("Kolja Lampe")
+    .url("https://fdsext.gemalto.com/filesSN/Index/79cca6a24fcc41c48a6dc31162a9ee7e?sid=6aded7501b842c50f12064606e4bcbaf")
```

Comment:
```suggestion
    .url("https://fdsext.gemalto.com/filesSN/Index/79cca6a24fcc41c48a6dc31162a9ee7e?sid=6aded7501b842c50f12064606e4bcbaf.exe")
```
The problem is that we try to infer the extension:
https://github.com/PhoenicisOrg/phoenicis/blob/dc91631f4b0e03142952d761600838de6cb53cb8/phoenicis-tools/src/main/java/org/phoenicis/tools/files/FileUtilities.java#L270